docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -97,22 +112,7 @@ When linking to another dashboard that uses template variables, select variable
 
 If you want to add all of the current dashboard's variables to the URL, then use `${__all_variables}`.
 
-## Data links
-
-Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization showed four servers, you could add a data link to one or two of them.
-
-The link itself is accessible in different ways depending on the visualization. For the Graph you need to click on a data point or line, for a panel like
-Stat, Gauge, or Bar Gauge you can click anywhere on the visualization to open the context menu.
-
-You can use variables in data links to send people to a detailed dashboard with preserved data filters. For example, you could use variables to specify a time range, series, and variable selection. For more information, refer to [Data link variables](#data-link-variables).
-
-### Typeahead suggestions
-
-When creating or updating a data link, press Cmd+Space or Ctrl+Space on your keyboard to open the typeahead suggestions to more easily add variables to your URL.
-
-{{< figure src="/static/img/docs/data_link_typeahead.png"  max-width= "800px" alt="Drop-down list with variable suggestions open from the URL field" >}}
-
-### Add a data link
+## Add a data link
 
 1. Hover over any part of the panel you want to which you want to add the data link to display the actions menu on the top right corner.
```

Comment:
```suggestion
1. Hover over any part of the panel to which you want to add the data link to display the actions menu in the top right corner.
```

Reply:
All good, this will get resolved on another PR that'll be merged into this one later. Just keeping my PR scopes really separate to make review easier for the team (my last PR was a bit of a nightmare, lol).